### PR TITLE
1.18

### PR DESCRIPTION
**Pull request by piuvas**
Sorry for 3 commits I was having trouble with IntelliJ

### CODE REVIEW

**Comment by Minenash** [comment]
Once I can get to a dev environment again, I'll test it and merge

**Comment by Minenash** [comment]
Hey guys, so sorry I haven't done anything with either PRs. Today Iupdated (by dep change or merge) most of my other mods, and this is one of the few left. Though, I'm not sure which of the two PRs to merge. Would either one of you be willing to maintain this project?
Despite it being one of my most downloaded mods, even more than my other mods, I have a hard time getting any will to do anything with this, for 3 major reasons:
- Me using a screen to sync up the next render for a screenshot has caused the following problems. But I don't know of a good alternative that doesn't require adding something to the main render loop to check for a variable. I mean 1 boolean check doesn't seem bad, but it just feels unnecessary.
- - Causing mods that don't check for hide gui flag, show up. easy for other mod authors to fix, but they shouldn't need to
- - Random breakage with Fabric API. Since 0.30, they added a new api that broke this mod, 'cause they made a false assumption. But didn't fix it "because use shouldn't be doing that anyways", and I think the workaround works, but I think I've gotten 1 or 2 bug reports where it still crashed because of it
- So, for this mod to work, I either have to mixin or replace ever screen that pops up between choosing a world/server to the world being generated. However with some mods, *despite having no code that would interfere with it*, it can still randomly show a dirt screen.

TL;DR: Despite being the most popular (not due to a modpack) mod of mine, having someone else work on it would be the best thing for this mod, as I just can't anymore

(just in case he doesn't get notified, @cm-360)

**Comment by spikespaz** [comment]
Superseded by #58
